docs/source/includes/gcloud_setup.rst

```diff
@@ -1 +1,24 @@
 Follow the Windows, Mac OS X or Linux instructions to install gcloud on your local machine: https://cloud.google.com/sdk/
+
```

Comment:
Thanks Matt !

Let me push back on this a tiny bit ;)

As someone who went through all of these things recently I can tell you
that its very frustrating that you can't clearly see the steps that need to
be done but rather have to navigate through the forest of links to
instruction that have links to further instructions and that sometimes
builds up a stack 4 level deep.

I also saw many 20%s forget to do gcloud auth login/project config stuff so
want to make sure its clearly spelled out.

Hence I very much wanted to inline everything here :)

So how about we keep the inlined version and we add a caveat and say this:

*Here are the steps to install gcloud on your local machine. They are based
on the documentation at  https://cloud.google.com/sdk/
<https://cloud.google.com/sdk/>, you can check this site in case the
instructions have changed.*

Wdyt ?
   Ilia.

On Tue, Oct 27, 2015 at 11:41 PM, mbookman <notifications@github.com> wrote:

> In docs/source/includes/gcloud_setup.rst
> <https://github.com/googlegenomics/readthedocs/pull/81#discussion_r43215853>
> :
>
> > @@ -1 +1,24 @@
> >  Follow the Windows, Mac OS X or Linux instructions to install gcloud on your local machine: https://cloud.google.com/sdk/
> > +
>
> It would be great if we could directly in-line the instructions for
> installing gcloud, but I'd rather not have to keep up with changes to the
> setup instructions, which do change slightly over time. Sending users to
> the cloud/sdk site is fine IMO.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/readthedocs/pull/81/files#r43215853>.
>